tests/common.py

```diff
@@ -1646,3 +1647,47 @@ def extract_stack_to_frame(extract_stack: list[Mock]) -> FrameType:
         current_frame = next_frame
 
     return top_frame
+
+
+def setup_test_component_platform(
+    hass: HomeAssistant,
+    domain: str,
+    entities: Sequence[MockEntity | MockToggleEntity],
```

Comment:
Why not just a list of entities, why do we need to restrict it?
```suggestion
    entities: List[Entity],
```